.dependabot/config.yml

```diff
@@ -10,3 +10,13 @@ update_configs:
       - match:
           update_type: "all"
           dependency_type: "direct"
+  - package_manager: "javascript"
+    directory: "/api_tests"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+      - "automagically-mergify-this"
+    allowed_updates:
+      - match:
+          update_type: "all"
+          dependency_type: "direct"
```

Comment:
:scream: Missing newline at end of file.

Reply:
too late! next time, just fix yourself or do a suggestion please.